remoulade/api/app/__init__.py

```diff
@@ -0,0 +1,27 @@
+""" Creating the instance of the app """
+from flask import Flask
+
+from remoulade.state import State
+from typing import Dict, List
+from .config import Config
+
+
+class MessageAdmin:
+    def __init__(self):
+        self.app = Flask(__name__)
+        self.app.config.from_object(Config)
+
+    def get_app(self):
+        return self.app
+
+    def get(self, message_id) -> Dict[str, str]:
+        res = self.app.config["backend"].get_state(message_id)
```

Comment:
I would prefer that you get the state backend via the remouade broker.
check `get_cancel_backend` and do something similar for state.